### PR TITLE
#162949065 Integrate Travis CI with Readme Badge

### PR DESCRIPTION
**Pull request by johnwayodi**
#### What does this PR do?
Integrates Travis CI tool with the repository

#### Description of Task to be completed?
Update the README.md with the travis-ci build badge.
Integrate the repository with [travis-ci.org](https://travis-ci.org/)

#### How should this be manually tested?
1. Open the project repository on github here : [ah-robotics](https://github.com/andela/ah-robotics)
3. Switch to branch `ch-travis-integration-162949065`
4. Scroll down to the _**README**_ section
5. Click on the **_build_** badge to see the build info on travis-ci website.
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#162949065](https://www.pivotaltracker.com/story/show/162949065)

#### Screenshot
screen shot 2019-01-08 at 12 24 00 am

#### Questions:
Feel free to give feedback and raise any questions.

### CODE REVIEW

**Comment by sharkdevs** [comment]
The CI badge is integrated fine and existing in the readme of the develop branch.  Question, why is it not pointing to chore branch you are working on?

**Comment by johnwayodi** [comment]
> The CI badge is integrated fine and existing in the readme of the develop branch. Question, why is it not pointing to chore branch you are working on?

@sharkdevs by default, Travis will start tracking from the default branch which is the develop branch, after I have merged my changes to develop is when you'll be able to see the build status of the various branches thereafter.

**Comment by Paulvitalis200** [comment]
Good integration of the Travis CI badge. Mine is just to tell you to ensure the *.travis.yml* file has been configured appropriately. The part where the python versions have been written down, should we only use **3.7** or list all the python versions as you have done?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+language: python
+
+python:
+  - "3.5"
+  - "3.6"
+  - "3.7"
```
Comment (waracci) [inline .travis.yml]: Why are several versions of python listed here? Isn't it supposed to have a single Python version only - the one used during development.

Comment (johnwayodi) [reply .travis.yml]: @waracci @Paulstar200 thanks for the highlight. I have updated the travis file to use only python3.6.

**Review by waracci — COMMENTED** [review]
I see you updated the python version to 3.6. 👍 
> Everything else looks okay to me.

**Review by Paulvitalis200 — APPROVED** [review]

**Comment by Teatoller** [comment]
Hello John,

You have incorporated a smart way to give a user a quick glimpse as to the integration of the application with Travis.
I would like to make a few pointers;

a). The TTL mentioned that for the reviewers to be able to give feedback they should be able to first clone the repository onto their machines. You need to revise your PR to have a step-by-step guide on how to go about this. 
b). The Travis badge is showing "Build Unknown" are we able to do better to have either a "Build Passing" or "Build Failing"
c.) I have seen the earlier reviewers point out the python version to be used. If you follow this link 
```https://github.com/andela/ah-robotics/commit/83a4b3b6cec50e9f9c473b915b2bf954426755e4```
It seems indicative that _the version that we ought to work with is Python 3.5.2_

#### Screenshoot

screenshot 2019-01-09 at 06 08 29

Do consider these inputs to enrich the PR.

*Regards*
*Steven Ennis*

**Comment by johnwayodi** [comment]
@Teatoller thanks for the comment, regarding the python version, we decided to use python 3.6 after I consulted some of the team members. I'll have to ask the TTL for more clarification on that.

> a). The TTL mentioned that for the reviewers to be able to give feedback they should be able to first clone the repository onto their machines. You need to revise your PR to have a step-by-step guide on how to go about this.

I haven't added any functionality to the project, the aim of this PR was to only add a Travis badge to the Readme, therefore cloning the branch and testing it locally will be too much overhead. You can just test if it works by heading to the remote branch and clicking the badge to see if it links to travis-ci.org.

> b). The Travis badge is showing "Build Unknown" are we able to do better to have either a "Build Passing" or "Build Failing"

The badge status is **currently** unknown because the badge points to the develop branch, once this branch is merged to develop, it will update to either passing or failing as you noted.

**Pull request merged**